src/cases/containers/query-management-container/query-management-container.component.html

```diff
@@ -69,14 +69,23 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
     </ccd-query-check-your-answers>
 
     <div>
-      <ng-container *ngIf="queryItem; else raiseQueryBackLink">
-        <a href="javascript:void(0)" (click)="goToQueryList()" class="govuk-back-link govuk-!-margin-bottom-0">
-          {{ 'Cancel and return to query list' | rpxTranslate }}
-        </a>
+      <ng-container *ngIf="queryCreateContext !== queryCreateContextEnum.RESPOND; else respondQuery">
```

Comment:
I think this is ngIf else wrapped in another ngIf else is very confusing; can we use ngSwitch instead?